packages/storybook/src/stories/Test.stories.mdx

```diff
@@ -0,0 +1,18 @@
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
+import { Button } from "./Button";
+
+<Meta title="Test" />
+
+Here's some _markdown_! Test
+
+# Preview
+
+<Preview>
+  <Story name="button">
+    <Button>hello</Button>
+  </Story>
+</Preview>
+
+# Props
+
+<Props of={Button} />
```

Comment:
This looks strange, I would imagine that stories would usually be a `tsx` type of file. 
Was it autogenerated or did we add it ouselves?

Reply:
That was a "Test" I added to check the `.mdx` files were being displayed correctly.